src/platform_impl/web/window.rs

```diff
@@ -277,6 +277,12 @@ impl Window {
         // Intentionally a no-op, as canvases cannot be 'minimized'
     }
 
+    #[inline]
+    pub fn is_minimized(&self) -> Option<bool> {
+        // Intentionally a no-op, as canvases cannot be 'minimized'
+        None
```

Comment:
It means that it's always `Some(false)`? The comment about a `no-op` doesn't make any sense.